haiku-apps/bedc/bedc-0.5.0~git.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A P2P file sharing client for the Direct Connect Protocol"
+DESCRIPTION="BeDC is a BeOS file sharing client compatible \
+with the Direct Connect protocol.
+Currently only chatting works, not filesharing/downloading."
+HOMEPAGE="https://github.com/HaikuArchives/BeDC"
+COPYRIGHT="2002 The BeDC team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+srcGitRev="cf2f67fba574e934a06026c66b4f4298470716c8"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9a4a632fa21cfd7d1b9adfca3179356d81e26308d67a0e6dcbfdd4546bb2d18c"
+SOURCE_FILENAME="BeDC-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="BeDC-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	BeDC = $portVersion
+	app:BeDC = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
```

Comment:
```diff
 	"
 
+defineDebugInfoPackage bedc \
+	$appsDir/BDirectConnect
+
 BUILD()
```
Shall we add a debuginfo package?

(I've just tested it and it creates a real (non-fake) debuginfo package.)
Thanks!

Reply:
Oops, that's a bad idea.
It strips the icon.